### PR TITLE
App usages over 16 years (EXPOSUREAPP-1781)

### PR DESCRIPTION
**Pull request by apopovsap**
https://jira.itc.sap.com/browse/EXPOSUREAPP-1781#add-comment

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/include_16_years.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        <import type="de.rki.coronawarnapp.util.formatter.FormatterHelper" />
+
+        <import type="de.rki.coronawarnapp.util.formatter.FormatterAccessibilityHelper" />
+
+        <import type="android.view.View" />
+
+        <variable
+            name="tracingViewModel"
+            type="de.rki.coronawarnapp.ui.viewmodel.TracingViewModel" />
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/layout/include_16_years.xml:15]: Variable seems unused

File: Corona-Warn-App/src/main/res/layout/include_16_years.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <import type="de.rki.coronawarnapp.util.formatter.FormatterHelper" />
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/layout/include_16_years.xml:7]: Formatters seem unused

File: Corona-Warn-App/src/main/res/layout/include_16_years.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+        <import type="de.rki.coronawarnapp.util.formatter.FormatterAccessibilityHelper" />
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/layout/include_16_years.xml:9]: Formatters seem unused

File: Corona-Warn-App/src/main/res/layout/include_16_years.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+        <import type="android.view.View" />
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/layout/include_16_years.xml:11]: View seems unused

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -479,6 +479,16 @@
     <!-- XACT: onboarding(notifications) - illustraction description, header image -->
     <string name="onboarding_notifications_illustration_description">"A woman receives a notification from her Corona-Warn-App."</string>
 
+    <!-- ####################################
+              Onboarding sixteen include
+    ###################################### -->
+
+    <!-- XACT: onboarding(sixteen) title -->
+    <string name="sixteen_title_text">"App Nutzung ab 16 Jahren"</string>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/strings.xml:487]: Default strings should be empty

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -479,6 +479,16 @@
...
+
+    <!-- XACT: onboarding(sixteen) title -->
+    <string name="sixteen_description_text">"Die App richtet sich an Personen, die sich in Deutschland aufhalten und mindestens 16 Jahre alt sind."</string>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/strings.xml:490]: Default strings should be empty

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 7]
```diff
@@ -198,6 +205,14 @@
         <item name="android:textColor">@color/colorTextPrimary1</item>
     </style>
 
+    <style name="headline6Sixteen" parent="@style/TextAppearance.MaterialComponents.Headline6">
+        <item name="android:textColor">@color/colorTextEmphasizedButton</item>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/styles.xml:209]: Is there a non button reference you could use?

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 8]
```diff
@@ -198,6 +205,14 @@
...
+    </style>
+
+    <style name="subtitleSixteen" parent="@style/TextAppearance.MaterialComponents.Subtitle1">
+        <item name="android:textColor">@color/colorTextEmphasizedButton</item>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/styles.xml:213]: Is there a non button reference you could use?

**Review by kolyaopahle — CHANGES_REQUESTED** [review]
LGTM 👍 i made some small linting suggestions other than those im quite happy

**Review by kolyaopahle — APPROVED** [review]

**Comment by jakobmoellersap** [comment]
https://jira.itc.sap.com/browse/EXPOSUREAPP-2092

**Pull request merged**